crate/cli/src/actions/kms/secret_data/create_secret.rs

```diff
@@ -0,0 +1,131 @@
+use clap::Parser;
+use cosmian_kmip::kmip_2_1::{
+    kmip_attributes::Attributes, requests::create_secret_data_kmip_object,
+};
+use cosmian_kms_client::{
+    KmsClient,
+    kmip_2_1::{kmip_types::UniqueIdentifier, requests::import_object_request},
+    reexport::cosmian_kms_client_utils::create_utils::SecretDataType,
+};
+use zeroize::Zeroizing;
+
+use crate::{
+    actions::kms::console,
+    error::result::{KmsCliResult, KmsCliResultHelper},
+    reexport::cosmian_kms_client::kmip_2_1::requests::secret_data_create_request,
+};
+
+/// Create a new secret data
+///
+/// Tags can later be used to retrieve the key. Tags are optional.
+#[derive(Parser, Default)]
+#[clap(verbatim_doc_comment)]
+pub struct CreateKeyAction {
```

Comment:
This should probably be named `CreateSecretDataAction` since it only creates that